hatchways/team-old-fashioned
# Feature: Sockets Implementation for Notifications

### What this PR does (required):
- Updates notifications (both dropdown and Notifications page) when user submits a design.

### Screenshots / Videos (required):
https://user-images.githubusercontent.com/78225194/127560785-6e9ba19a-4f0c-465d-9d0c-bebd470da837.mp4

### Any information needed to test this feature (required):
- Check whether notifications are available upon logging in.
- Upload/submit an image then access the Notifications dropdown and Notifications page. These should be updated without having to refresh the page.

### Any issues with the current functionality (optional):
- Triggering notification creation for new messages is not yet implemented while Messaging is not yet available.
- May have to identify notification receiver when emitting "notification created" from the server.
